### ISSUE TITLE
About"logits = maurice_model.localDepthMap(test_image, coarse, keep_conv, keep_hidden)"

### ISSUE THREAD

**Issue by hahapp** [KomodoFactory/Monocular-Depth-Estimation#4]
Hi, in these code:
---------------------------------------------------------------------------------------------
        coarse = maurice_model.globalDepthMap(test_image, keep_conv, trainable=False)
        logits = maurice_model.localDepthMap(test_image, coarse, keep_conv, keep_hidden)  
------------------------------------------------------------------------------------------
What do keep_conv and keep_hidden mean?
thanks,
